### PR TITLE
Edited Ridgeline.Rmd

### PR DESCRIPTION
**Pull request by nehasaraf1994**
Edited with the changes you asked. Just did not understand the third point of legends.

### CODE REVIEW

**Comment by nehasaraf1994** [comment]
Sorry for another pull request but did not understand to do in the same pull request

**Comment by zachbogart** [comment]
(Updated version of PR #152)

**Comment by nehasaraf1994** [comment]
Do I have to do anything to merge it? Or my part is done?

File: RidgelinePlots.Rmd [hunk 1]
```diff
@@ -0,0 +1,136 @@
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library("ggridges")
+library("tidyverse")
+library("cluster")
+```
+# Chart : Ridgeline Plots {ridgeline}
```
Comment (zachbogart) [inline RidgelinePlots.Rmd]:
```suggestion
# Chart : Ridgeline Plots {#ridgeline}
```

**Comment by zachbogart** [comment]
Looks good. Two things to check it is ready to publish:
- Please sync your fork (https://help.github.com/articles/syncing-a-fork/)
- Then update the _bookdown.yml file to include your Ridgeline.Rmd file (put it in between scatterplot and dates)

**Pull request merged**